### PR TITLE
CE-3569: Set infobox to default state on every dialog open

### PR DESCRIPTION
**Pull request by lukaszkonieczny**
## Links

* https://wikia-inc.atlassian.net/browse/CE-3569

## Description

We want to open dialog with infobox builder always in default state

## Reviewers

@Wikia/community-engineering @RafLeszczynski @Wikia/x-wing

### CODE REVIEW

**Comment by adamkarminski** [comment]
@lukaszkonieczny - Have seen the failed checks?

File: front/main/app/components/infobox-builder.js [hunk 1]
```diff
@@ -290,10 +290,10 @@ export default Ember.Component.extend(
 					label: 'save-successful'
 				});
 
-				this.setProperties({
-					isLoading: false,
-					showSuccess: true
```
Comment (adamkarminski) [inline front/main/app/components/infobox-builder.js]: you could probably go with:
```isLoading: false,
showSuccess: !this.get('isVEContext')```

File: front/main/app/routes/infobox-builder.js [hunk 2]
```diff
@@ -336,6 +329,11 @@ export default Ember.Route.extend(ConfirmationMixin, {
 	 * @returns {void}
 	 */
 	setVEContext(isVEContext = false) {
-		this.controllerFor('infobox-builder').set('isVEContext', isVEContext);
+		const controller = this.controllerFor('infobox-builder');
+
+		controller.set('isVEContext', isVEContext);
+		if (isVEContext) {
+			controller.set('isDirty', false);
```
Comment (adamkarminski) [inline front/main/app/routes/infobox-builder.js]: Please check if you can go with `controller.set('isDirty', !isVEContext)` :)

Comment (lukaszkonieczny) [inline front/main/app/routes/infobox-builder.js]: This one will leave as it is.

Comment (adamkarminski) [inline front/main/app/routes/infobox-builder.js]: 👍

**Comment by adamkarminski** [comment]
LGTM

**Pull request merged**